lib/benchee.ex

```diff
@@ -33,6 +33,7 @@ defmodule Benchee do
 
   defp do_run(jobs, config) do
     jobs
+    |> normalize_names
```

Comment:
heyho!

Don't think this is the best place. It works for the `Benchee.run` interface but for the alternative `Benchee.init |> Benchee.benchmark("...", fn -> .. end) |> Benchee.measure`

e.g. the function here: https://github.com/PragTob/benchee/blob/master/lib/benchee/benchmark.ex#L19-L27

So, there's 2 possible entry points for benchmarks.

I think what I'd aim for is making the normalization in `Benchee.Benchmark.benchmark` and instead of directly passing on the map we get here let it pass through

```
Enum.each(job, fn(key, value) -> Benchee.Benchmark.benchmark(suite, key, value))

```

Or something to that effect, that way both pass through the normalization before ending up in the `:jobs` key of suite.

What do you think? :)